app.js

```diff
@@ -20,9 +20,27 @@ app.allUsers = [];
 // Array of usernames queued up to be assigned to upcoming new issues
 app.queuedUsers = [];
 
-// When receiving a POST request from Sentry:
-app.post("/", async function(request, response) {
+const errorWrapper = fn => {
+  return async (req, res, next) => {
+    try {
+      await fn(req, res, next);
+    } catch(err) {
+      const errorId = sentry.captureException(err);
+
+      res.statusCode = 500;
```

Comment:
@nolachen better to use `res.status(500)` here instead of setting the status code in the response like this.